internal/utils.go

```diff
@@ -86,9 +86,11 @@ func getManifests(policyConf *types.PolicyConfig) ([]map[string]interface{}, err
 	return manifests, nil
 }
 
-// getPolicyTemplates generates the policy templates for the ConfigurationPolicy manifests
```

Comment:
Please keep this comment structure that starts with a short sentence describing the function that starts with the function name. Your other comments can go after it.

Reply:
Here is a reference by the way:
https://go.dev/blog/godoc